docs/source/developers/guide/documentation.rst

```diff
@@ -27,4 +27,65 @@
 
 **************************
 Helping with documentation
-**************************
\ No newline at end of file
+**************************
+
+
+A great way to contribute to the project is to improve
+documentation. 
+
+If you are an Arrow user and you found some docs to be
+incomplete or inaccurate, share your hard-earned knowledge
+with the rest of the community.
+
+If you haven't come across an error in the documentation
+it is hard to find one and going through the docs in search
+for corrections is not a good way to go. In this case you can
```

Comment:
This first sentence is a bit hard to follow. Maybe adding a comma after the first line would already help. 

Maybe just saying "If you didn't come across something to improve in the documentation itself, you can search for an issue ..." is sufficient?

Reply:
Yes, you are quite right. It's a confusing sentence. Will correct, thanks!